### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by delirious-lettuce**
This PR fixes numerous typos throughout the project.

### CODE REVIEW

File: src/unity/python/turicreate/toolkits/graph_analytics/pagerank.py [hunk 1]
```diff
@@ -35,7 +35,7 @@ class PagerankModel(_ModelBase):
     +-------------------+-----------------------------------------------------------+
     | Field             | Description                                               |
     +===================+===========================================================+
-    | reset_probability | The probablity of random jumps to any node in the graph   |
+    | reset_probability | The probability of random jumps to any node in the graph   |
```
Comment (TobyRoseman) [inline src/unity/python/turicreate/toolkits/graph_analytics/pagerank.py]: Fixing it so the right side of the box lines up would be nice. You should just need to remove a space after "graph".

File: src/unity/python/turicreate/toolkits/regression/boosted_trees_regression.py [hunk 2]
```diff
@@ -129,7 +129,7 @@ def _get(self, field):
         +-------------------------+--------------------------------------------------------------------------------+
         | trees_json              | Tree encoded using JSON                                                        |
         +-------------------------+--------------------------------------------------------------------------------+
-        | valiation_error         | Error on validation data                                                       |
+        | validation_error         | Error on validation data                                                       |
```
Comment (TobyRoseman) [inline src/unity/python/turicreate/toolkits/regression/boosted_trees_regression.py]: same as previous comment

File: src/unity/python/turicreate/toolkits/regression/decision_tree_regression.py [hunk 3]
```diff
@@ -151,7 +151,7 @@ def _get(self, field):
         +-------------------------+--------------------------------------------------------------------------------+
         | trees_json              | Tree encoded using JSON                                                        |
         +-------------------------+--------------------------------------------------------------------------------+
-        | valiation_error         | Error on validation data                                                       |
+        | validation_error         | Error on validation data                                                       |
```
Comment (TobyRoseman) [inline src/unity/python/turicreate/toolkits/regression/decision_tree_regression.py]: same here too.

File: src/unity/python/turicreate/toolkits/regression/random_forest_regression.py [hunk 4]
```diff
@@ -155,7 +155,7 @@ def _get(self, field):
         +-------------------------+--------------------------------------------------------------------------------+
         | trees_json              | Tree encoded using JSON                                                        |
         +-------------------------+--------------------------------------------------------------------------------+
-        | valiation_error         | Error on validation data                                                       |
+        | validation_error         | Error on validation data                                                       |
```
Comment (TobyRoseman) [inline src/unity/python/turicreate/toolkits/regression/random_forest_regression.py]: one more quick fix

**Comment by delirious-lettuce** [comment]
@TobyRoseman ,

Oops! I should have caught those formatting issues, I'll try and blame it on working late haha

I will fix those issues ASAP.

**Comment by delirious-lettuce** [comment]
@TobyRoseman ,

Should be all fixed now!

**Comment by TobyRoseman** [comment]
@delirious-lettuce changes look good and I've verified unit tests pass, but look like there's now another git conflict.

**Comment by znation** [comment]
@delirious-lettuce The changes are conflicting with what's currently in master. Would you mind following the instructions [here](https://help.github.com/articles/about-git-rebase/) to rebase the change against the latest master so it will merge cleanly? While you're at it, please feel free to condense this change into fewer commits, but that's optional. Once you've rebased I'm guessing you will need to force push to the PR branch, since your local history will have been rewritten. Thanks!

**Comment by delirious-lettuce** [comment]
@znation ,

Sorry for the delayed response, I'm going to try to rebase and push right now.

**Comment by delirious-lettuce** [comment]
@znation ,

I made a mess of this one so I opened a new branch, squashed these commits into one and pushed these typo fixes there (#106 ). Sorry for the confusion!

**Comment by znation** [comment]
No problem @delirious-lettuce, thanks for the fixes!

**Pull request closed without merging**